ios/Scripts/generate_app_icons.py

```
@@ -0,0 +1,155 @@
+
+        except Exception as e:
+            print(f"❌ Ошибка при создании {filename}: {e}")
+
+    print(f"\n🎯 Готово! Создано {success_count}/{total_count} иконок")
+    return success_count == total_count
+
+
+if __name__ == "__main__":
+    # Проверяем наличие PIL
+    try:
+        from PIL import Image, ImageDraw
```

Flake8 (7.2.0)

[error] redefinition of unused 'Image' from line 9

(F811)